Movie_Ticket_Booking/src/main/java/com/org/Movie_Ticket_Booking/controller/manager/RoomController.java

```diff
@@ -16,22 +17,23 @@
 import java.util.Set;
 
 @Controller
-@RequestMapping("/room")
-public class RoomController {
+public class RoomController extends ManagerController{
     private RoomService roomService;
 
     @Autowired
     public RoomController(RoomService roomService){
         this.roomService=roomService;
     }
 
-    @GetMapping("/{roomId}/seats")
+    @GetMapping("/room/{roomId}/seats")
```

Comment:
```suggestion
    @GetMapping("/rooms/{roomId}/seats")
```

Cơ mà xóa ko khai báo `/rooms` lên trên chỗ define class này nhỉ
các action trong coller này đều sẽ start với /rooms

Reply:
Chị ơi em đã fix hết comment rồi ạ